src/editor/editor.instance.ts

```diff
@@ -168,6 +172,7 @@ export const clearEditorDecorations = () => {
 
 monaco.languages.registerHoverProvider('yaml', {
   provideHover: (model, position) => {
+    trackEvent('editor/hover');
```

Comment:
perhaps we should track this after checking if we actually have something to show?

Reply:
Since Monaco could provide it's own hovers, for YAML / Kubernetes schema errors, which are not created by us, I it makes more sense to send that event for any hover.
I added a comment about this to the code

Reply:
agree - I thought this method was called by Monaco for any hover opportunity - it's first after the editorHovers.filter that we know if there is actually something to show.. !?

Reply:
I added a "type" property for our own hovers and now check if I can send that or not